python/cudf/cudf/core/column/lists.py

```diff
@@ -178,6 +178,32 @@ def __init__(self, column, parent=None):
             )
         super().__init__(column=column, parent=parent)
 
+    def contains(self, key, _dtype):
```

Comment:
Why are we passing `_dtype` here? We should have all of the information needed in order to type `key` if someone passes something that isn't strongly typed.